tests/run/sys_path_globbed_T5942.srctree

```diff
@@ -0,0 +1,33 @@
+PYTHON setup.py build_ext --inplace
+PYTHON -c "from tglob import mytest; mytest()"
+
+######## __init__.py ########
+
+######## tglob.pyx ########
+def mytest():
+    assert True
+######## setup.py ########
+import sys
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup, Extension
+
+sys.path.append("[Hello-World]*?")
+
+try:
+    from __builtin__ import reload
+except ImportError:
+    from importlib import reload
+reload(sys)  # Magic
+
+ext_modules = [
+    Extension(
+        "tglob",
+        sources=["tglob.pyx"],
+    )
+]
+
+setup(
+    ext_modules=cythonize(ext_modules, language_level="3"),
+)
+
```

Comment:
```suggestion
setup(
    ext_modules=cythonize(["tglob.pyx"], language_level="3"),
)

```

Reply:
This suggested change fails the test. Working on a fix.

Reply:
I made it more compact without changing the semantics. If the semantics need to be changed and the fix passes the test, let me know what the changes are. Thanks for your help!

Reply:
Ah, you added an `__init__.py` file to your test source tree. Remove it, your test isn't in a package.